source/advanced-topics.adoc

```diff
@@ -1143,3 +1143,77 @@ The `%ifos` conditional is used to control processing based on the operating sys
   %endif
 
 All the contents of the SPEC file between `%ifos` and `%endif` are processed only if the build was done on a Linux system.
+
+[id="Packaging-of-Python-RPMs"]
+=== Packaging of Python RPMs
+
+[id="Benefits-of-packaging-Python-packages-as-RPMs"]
+==== Benefits of packaging Python packages as RPMs
+
+Benefits of packaging Python packages as RPMs are the same as for any other RPM package. More can be found in the chapter xref:Why-Package-Software-with-RPM[].
```

Comment:
I'd maybe add some thinking about comparison with working with native python packages, like:
In comparison to native python packages from Python Package Index, RPM allows to specify dependencies on other RPMs, that are not necessary Python packages, RPM also allows to use cryptographic signing to verify content, and deliver packages that are integrated and tested ahead with the rest of the system. On the other hand RPM does not allow users to install Python software as a non-root user or to an arbitrary location.

Reply:
Thank you, good idea.